pkg/config/config.go

```diff
@@ -204,6 +204,9 @@ func (c *config) Current() (types.BacalhauConfig, error) {
 	if err := c.base.Unmarshal(&out, DecoderHook); err != nil {
 		return types.BacalhauConfig{}, err
 	}
+	if out.Repo == "" {
+		out.Repo = viper.GetString("repo")
+	}
```

Comment:
can you clarify this?

Reply:
added a comment in latest changes, in the event a user has provided a config without a data dir we need to set it here based on what the repo was initialized with.